docker/run.sh

```diff
@@ -15,20 +15,12 @@ if [ $# -eq 2 ]; then
   fi
 fi
 
-
-# build docker container
-npm install & npm run build
-docker build -f docker/Dockerfile . --tag=vectara-answer:latest
-
-# run the docker with the right configuration
-docker rm vanswer -f 2> /dev/null
-
 # If we used legacy configs (prepared via Python and using queries JSON file),
 # use the Docker command to forward the queries.json path to the container.
 if [ -z "$correct_path" ]; then
-  docker run --platform=linux/amd64 -d --env-file .env --name vanswer -p 127.0.0.1:80:3000/tcp vectara-answer
+  docker compose up -d
```

Comment:
This seems to still be using "docker compose" - let's switch back to just using "docker run" as discussed in the comment?